README.md

```diff
@@ -63,7 +63,9 @@ directly in the web browser with <a href="https://github.com/feross/webtorrent">
   </a>
 </p>
 
-<p align="center">
+## Getting Started
+
+<p align="left">
   <strong><a title="Website" target="_blank" href="https://joinpeertube.org">Website</a> |
```

Comment:
If you remove the p tag, you can use markdown here. If you don't know how to do, I'll do it mysefl :)